### PR TITLE
Update README with initial information

### PR DESCRIPTION
**Pull request by hillaryj**
#  #

## 🗣 Description ##

Replaces skeleton information with useful information.

## 🧪 Testing ##

Documentation only, no testing required other than linting.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,22 +2,20 @@
...
+  - How do I make an account?
+  - How do I get access?
+  - How do I make a repository?
+  - Content guides & best practices
+
+## Developer resources ##
+
+We have a [cisagov development-guide](www.github.com/cisagov/development-guide)
+repository, which contains coding standards, how to set up a development
+environment, and other information
```
Comment (jsf9k) [inline README.md]:
```suggestion
We have a [cisagov development-guide](www.github.com/cisagov/development-guide)
repository, which contains coding standards, steps for setting up a development
environment, and other information.
```

File: README.md [hunk 2]
```diff
@@ -2,22 +2,20 @@
...
-In many cases you will instead want to use one of the more specific
-skeleton projects derived from this one.
-
-## New Repositories from a Skeleton ##
-
-Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
-for step-by-step instructions on how to start a new repository from
-a skeleton. This will save you time and effort when configuring a
-new repository!
+Welcome to `cisagov`, the GitHub organization for CISA.
```
Comment (jsf9k) [inline README.md]:
```suggestion
Welcome to `cisagov`, the GitHub organization for the Cybersecurity and Infrastructure Security Agency (CISA).
```

File: README.md [hunk 3]
```diff
@@ -2,22 +2,20 @@
...
+
+## Getting started with GitHub ##
+
+  - How do I make an account?
+  - How do I get access?
+  - How do I make a repository?
+  - Content guides & best practices
```
Comment (jsf9k) [inline README.md]: This should fix the linting errors:
```suggestion
- How do I make an account?
- How do I get access?
- How do I make a repository?
- Content guides & best practices
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
I made some suggestions, one of which is a fix for the failing pre-commit linters.

**Review by dav3r — APPROVED** [review]
Approved, assuming you make the changes requested by @jsf9k. 👍 👍

**Review by jsf9k — APPROVED** [review]

**Review by mcdonnnj — APPROVED** [review]
📜

**Pull request merged**